packages/frontend/src/utils/project/getDiagramImage.test.ts

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'earl'
+import mockFs from 'mock-fs'
+import path from 'path'
+
+import { getDiagramImage } from './getDiagramImage'
+
+describe(getDiagramImage.name, () => {
+  it('returns the correct path if the file exists', () => {
+    mockFs({
```

Comment:
Do we need this mockFs dependency? Can't we just save some temporary file and at the end delete it?